src/cli/audit_command.zig

```diff
@@ -0,0 +1,570 @@
+                                if (vuln.data == .e_object) {
+                                    const vulnerability = try parseVulnerability(allocator, package_name, vuln);
+
+                                    if (std.mem.eql(u8, vulnerability.severity, "low")) {
+                                        vuln_counts.low += 1;
+                                    } else if (std.mem.eql(u8, vulnerability.severity, "moderate")) {
+                                        vuln_counts.moderate += 1;
+                                    } else if (std.mem.eql(u8, vulnerability.severity, "high")) {
+                                        vuln_counts.high += 1;
+                                    } else if (std.mem.eql(u8, vulnerability.severity, "critical")) {
+                                        vuln_counts.critical += 1;
+                                    } else {
+                                        vuln_counts.moderate += 1;
+                                    }
+
+                                    try audit_result.all_vulnerabilities.append(vulnerability);
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        for (audit_result.all_vulnerabilities.items) |vulnerability| {
+            const paths = try findDependencyPaths(allocator, vulnerability.package_name, dependency_tree, pm);
+
+            const result = try audit_result.vulnerable_packages.getOrPut(vulnerability.package_name);
+            if (!result.found_existing) {
+                result.value_ptr.* = PackageInfo{
+                    .package_id = 0,
+                    .name = vulnerability.package_name,
+                    .version = vulnerability.vulnerable_versions,
+                    .vulnerabilities = std.ArrayList(VulnerabilityInfo).init(allocator),
+                    .dependents = paths,
+                };
+            }
+            try result.value_ptr.vulnerabilities.append(vulnerability);
+        }
+
+        var package_iter = audit_result.vulnerable_packages.iterator();
+        while (package_iter.next()) |entry| {
+            const package_info = entry.value_ptr;
+
+            if (package_info.vulnerabilities.items.len > 0) {
+                const main_vuln = package_info.vulnerabilities.items[0];
+
+                if (main_vuln.vulnerable_versions.len > 0) {
+                    Output.prettyln(comptime Output.prettyFmt("<red>{s}<r>  {s}", true), .{ main_vuln.package_name, main_vuln.vulnerable_versions });
+                } else {
+                    Output.prettyln(comptime Output.prettyFmt("<red>{s}<r>", true), .{main_vuln.package_name});
+                }
+
+                for (package_info.vulnerabilities.items) |vuln| {
+                    if (std.mem.eql(u8, vuln.severity, "critical")) {
+                        Output.prettyln(comptime Output.prettyFmt("Severity: <red>critical<r>", true), .{});
```

Comment:
```suggestion
                        Output.prettyln("Severity: <red>critical<r>", .{});
```
`Output.pretty*` should handle pretty formatting